### DISCUSSION TITLE
setting SMPT on self-host.

### DISCUSSION THREAD

**Discussion by art3moff** [chatwoot/chatwoot#7945] [Self hosted]
I try to set my self-hosted CW to use gmail as SMTP Server

To Reproduce

Install via Docker on Linux VM 
Apps already running and i had tried the innboxe feature for 3 different channel (Telegram, Email dan Website) and working well

Where can I see the logs that are responsible for this connection to find out what the error is? 
I use a corporate e-mail account. 

env conf  (SMTP conf only) :
![image](https://github.com/chatwoot/chatwoot/assets/130499042/59b8fee5-4873-4b98-8b8e-3e8d00075e3e)

Problem: does not hear emails about verification and password reset for agents. and gives an error connecting to the Woot Server.

![image](https://github.com/chatwoot/chatwoot/assets/130499042/379a8738-7b17-413b-b7c9-db2b4c2dc8f6)

**Comment by sojan-official** [comment]
We don't officially support Gmail as a provider for the SMTP configuration for outbound emails. Please use a provider like mailgun, SendGrid etc.  but if you like to debug the issue with your existing credentials, please follow 

https://www.chatwoot.com/docs/self-hosted/faq#step-2-validate-your-smtp-config-by-sending-an-email

**Reply by art3moff** [reply]
I use my own domain email. Not gmail. 

Following the documentation you attached above, this is the conclusion : 

I, [2023-09-22T06:59:13.762747 #814] INFO -- : Delivered mail 650d3b3caf16f_32e192883544@1d76e9365163.mail (5046.0ms)
/usr/local/lib/ruby/3.2.0/net/protocol.rb:229:in `rbuf_fill': Net::ReadTimeout with #<Socket:(closed)> (Net::ReadTimeout

It is worth noting that the container itself can both resolve the host and reach it)
